csunplugged/topics/management/commands/_CurriculumAreasLoader.py

```diff
@@ -56,10 +56,19 @@ def load(self):
                     "Curriculum Area"
                 )
 
+            curriculum_area_colour = curriculum_area_data.get("colour", None)
+            if curriculum_area_colour is None:
+                raise MissingRequiredFieldError(
+                    self.structure_file_path,
+                    ["colour"],
```

Comment:
check MissingRequiredFieldError.py, you actually have to specify *all* of the required fields, not just the one that is missing as this is the error message it gives:

> A {model} requires the following fields:
{fields}
One or more of these fields are missing.